### ISSUE TITLE
Question - Packaging debian apps

### ISSUE THREAD

**Issue by JohnMurga** [pelya/debian-noroot#16]
Hi,

First of all thank you for this awesome tool, it works well for me on all my ARM/Intel devices :tada: 

I was thinking of building the core system with no apps or window manager in such a way that I could then package X11 apps as ready to run APKs (auto-launch the app within the xserver).

A bit like this works :
https://sourceforge.net/p/x11-basic/discussion/2447828/thread/8af1f9d8/#64d8

I know that for each app the installed size would be kinda large (*and the need to download arch specific archives for app and run-time at first run*).

Do you have any thoughts on this, or should I just try to build debian-noroot and then get dissecting ? :wink: 

Thanks again

**Comment by pelya** [comment]
Yeah that's possible, look at Gimp Inkscape or Shared printer plugin apps.
You will still need at least xfwm4 to draw window borders, but the desktop
is not strictly necessary.
